src/languageservice/parser/jsonParser07.ts

```diff
@@ -580,12 +580,18 @@ export class JSONDocument {
     return null;
   }
 
-  public getMatchingSchemas(schema: JSONSchema, focusOffset = -1, exclude: ASTNode = null): IApplicableSchema[] {
+  public getMatchingSchemas(
```

Comment:
Can you add documentation for this function. It is very hard to understand what `doComplete` intends to do?